### PR TITLE
Hw 262 new mailing schedule step

### PR DESCRIPTION
**Pull request by meladawy**
Here is the changes : 
1- Define Wizard as a new crm component 
2- Change placeholder color for text fields 
3- Add 2px rounded corner for panels as @jamienovick suggestion

hw-262

Wizard Component HTMl Structure 

```html
<div class="crm_wizard">

    <div class="panel crm_wizard__title">
        <div class="panel-body">
            <ul class="nav nav-pills">
                <li class="active">
                    <a href="">
                        <span class="crm_wizard__title__number"> 1 </span>
                        Step 1
                    </a>
                </li>
                
                <li>
                    <a href="">
                        <span class="crm_wizard__title__number"> 2 </span> 
                        Step 2
                    </a>
                </li>
                
                <li>
                    <a href="">
                        <span class="crm_wizard__title__number"> 3 </span> 
                        Step 3
                    </a>
                </li>
            </ul>
        </div>
    </div>

    <div class="panel crm_wizard__body">
        <div class="panel-body">
            Any body content
        </div>
    </div>

    <div class="panel crm_wizard__footer">
        <div class="panel-body">
            <div class="row">
                <div class="col-md-6 text-left">
                    <button class="btn-danger btn">Action</button>
                </div>
                <div class="col-md-6 text-right">
                    <button class="btn-primary btn">Action</button>
                </div>
            </div>
        </div>
    </div>

</div>
```

Wizard Component Output

screen shot 2017-03-30 at 3 15 16 pm

### CODE REVIEW

File: scss/bootstrap/overrides/style/_panel.scss [hunk 1]
```diff
@@ -1,8 +1,10 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
 
+  border-radius: 2px;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_panel.scss]: in bootstrap i think there is a generic border-radius-related variable, shouldn't we use that?

File: scss/bootstrap/overrides/style/_forms.scss [hunk 2]
```diff
@@ -25,6 +25,30 @@ label {
   fieldset[disabled] & {
     color: $crm-darker-grayblue;
   }
+
+  &[type="text"] {
+    &::-webkit-input-placeholder {
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_forms.scss:30]: What's the need for all these browser-specific styles?

File: scss/bootstrap/overrides/style/_panel.scss [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
   border: none;
   margin-bottom: $grid-gutter-width;
+  overflow: hidden;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_panel.scss]: Why is this needed?

File: scss/bootstrap/components/_wizard.scss [hunk 4]
```diff
@@ -1,11 +1,11 @@
-.crmb-wizard {
-  &-body,
-  &-title {
+.crm_wizard {
+  &__body,
+  &__title {
     margin-bottom: 30px;
```
Comment (AkA84) [inline scss/bootstrap/components/_wizard.scss:4]: Does this mean that any `.crm_wizard` will have a bottom margin?

**Review by AkA84 — CHANGES_REQUESTED** [review]
@meladawy you are defining a new component (`crm_wizard`), so in the PR please showcase its markup and a screenshot of what the result is. Like you would have for example in http://getbootstrap.com/components/#dropdowns-headers

File: scss/bootstrap/overrides/style/_panel.scss (hunk 3 above)
Comment (meladawy) [reply scss/bootstrap/overrides/style/_panel.scss]: Border radius is not applied to panels without having overflow hidden. not sure if there is any better approach

File: scss/bootstrap/overrides/style/_forms.scss (hunk 2 above)
Comment (meladawy) [reply scss/bootstrap/overrides/style/_forms.scss:30]: Shoudln't we support all the browsers?

File: scss/bootstrap/components/_wizard.scss (hunk 4 above)
Comment (meladawy) [reply scss/bootstrap/components/_wizard.scss:4]: Yea this margin should be applied to any wizard. 

wizard_margin

File: scss/bootstrap/overrides/style/_panel.scss [hunk 5]
```diff
@@ -1,8 +1,9 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
-
+  border-radius: $border-radius-small;
   border: none;
   margin-bottom: $grid-gutter-width;
+  overflow: hidden;
```
Comment (meladawy) [inline scss/bootstrap/overrides/style/_panel.scss]: This overflow property required to get border-radius working...not sure if there is any better approach

File: scss/bootstrap/overrides/style/_forms.scss (hunk 2 above)
Comment (AkA84) [reply scss/bootstrap/overrides/style/_forms.scss:30]: apologies, i just realized that this being for the placeholder it requires vendor-specific selectors

File: scss/bootstrap/overrides/style/_forms.scss [hunk 6]
```diff
@@ -25,6 +25,30 @@ label {
...
+      /* WebKit, Blink, Edge */
+      color: $crm-dark-grayblue;
+    }
+
+    &:-moz-placeholder {
+      /* Mozilla Firefox 4 to 18 */
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_forms.scss]: this is not necessary, firefox is currently at v52

File: scss/bootstrap/overrides/style/_forms.scss [hunk 7]
```diff
@@ -25,6 +25,30 @@ label {
   fieldset[disabled] & {
     color: $crm-darker-grayblue;
   }
+
+  &[type="text"] {
+    &::-webkit-input-placeholder {
+      /* WebKit, Blink, Edge */
+      color: $crm-dark-grayblue;
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_forms.scss]: Remove all these comments, it's evident enough what browser each selector is for, it's in the prefix

File: scss/bootstrap/overrides/style/_panel.scss (hunk 5 above)
Comment (AkA84) [reply scss/bootstrap/overrides/style/_panel.scss]: yeah i think we need to discuss this

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: scss/bootstrap/overrides/style/_panel.scss (hunk 5 above)
Comment (meladawy) [reply scss/bootstrap/overrides/style/_panel.scss]: I removed the overflow hidden and replaced it with another border-radius in .panel-body

plz check this commit
https://github.com/compucorp/org.civicrm.shoreditch/pull/5/commits/2f1e119b5635437b3866ff1717d11edc78c27fca

File: scss/bootstrap/overrides/style/_forms.scss (hunk 7 above)
Comment (meladawy) [reply scss/bootstrap/overrides/style/_forms.scss]: Resolved 

https://github.com/compucorp/org.civicrm.shoreditch/pull/5/commits/a9b01d5fbb541333ceb876873f3544afcbfac9d2

File: scss/bootstrap/overrides/style/_forms.scss (hunk 6 above)
Comment (meladawy) [reply scss/bootstrap/overrides/style/_forms.scss]: Resolved 
https://github.com/compucorp/org.civicrm.shoreditch/pull/5/commits/a9b01d5fbb541333ceb876873f3544afcbfac9d2

**Comment by meladawy** [comment]
@AkA84 I removed radius from panels and added it to Wizard as per our yesterday discussion 
https://github.com/compucorp/org.civicrm.shoreditch/pull/5/commits/25198903dbd06147ffef1ab32d652eb6e88cf72a

**Review by AkA84 — APPROVED** [review]

**Pull request merged**